lib/gosu/patches.rb

```diff
@@ -57,6 +57,31 @@ def tick
   end
 end
 
+class Gosu::Image
+  def ==(img)
```

Comment:
I don't think this should go into the public Gosu interface as `to_blob` will throw an exception for macros; and even if `to_blob` works on macros in the future (based on `render{}`), I'm not sure if a macro should be equal to an image just because it renders to the same pixel data at a given resolution.
Explicitly comparing `to_blob` avoids all of these issues.

Reply:
(And I realize that this means we should also kick `Image#==` out of `test_helper.rb`)